code/modules/projectiles/projectile.dm

```diff
@@ -191,8 +193,11 @@
 			if(hitsound)
 				var/volume = vol_by_damage()
 				playsound(loc, hitsound, volume, TRUE, -1)
-			L.visible_message("<span class='danger'>[L] is hit by \a [src][organ_hit_text]!</span>", \
-					"<span class='userdanger'>You're hit by \a [src][organ_hit_text]!</span>", null, COMBAT_MESSAGE_RANGE)
+			if(!L.message_timeouts || L.message_timeouts <= world.time)

```

Comment:
`if(!L.message_timeouts || !L.message_timeouts[type] || L.message_timeouts[type] <= world.time)`

Reply:
just (type)? nothing else?

Reply:
That's what you set with that LAZYSET as key so yes.